src/export/ExportPCM.cpp

```diff
@@ -33,6 +33,7 @@
 #include "../widgets/ProgressDialog.h"
 #include "../widgets/wxWidgetsWindowPlacement.h"
 #include "wxFileNameWrapper.h"
+#include "ExportFileDialog.h"
```

Comment:
I thought this was unnecessary for the build, but now I see it's needed for `AUDACITY_FILE_SUFFIX_EVENT`

That is one place where I omitted to change wxEvents to Observer messages, but then this is a case of wxWidgets dialogs communicating to each other.